### ISSUE TITLE
[Question] Will this keep forceloaded chunks loaded after server restart?

### ISSUE THREAD

**Issue by Jacob1101** [OffKilterMC/ChunkLoaderPersistence#4]
I'm too stoopid to understand the description of the mod but here's my issue:

When I forceload chunks using /forceload command, it's like the server forgets after a restart.

Is this the mod to fix this issue?

**Comment by OffKilterMC** [comment]
I don't think so. This is for hand-built chunk loaders to keep their portal tickets alive. 

I am surprised that /forceload doesn't stick around.  It does stick around on a single-player world. I'm surprised the server would work differently.
